FlightControlWeb/Controllers/FlightPlanController.cs

```diff
@@ -14,15 +14,15 @@ namespace FlightControlWeb.Controllers
     public class FlightPlanController : ControllerBase
     {
         private readonly DBInteractor db;
-        private readonly IFlightPlanManager Manager = new FlightPlanManager(new DBInteractor());
```

Comment:
I think this should be also in the constructor, like the DBInteractor. I think that the dependency injector is responsible for creating the instance and pass it to the constructor